components/eamxx/src/physics/p3/p3_functions.hpp

```diff
@@ -107,6 +107,13 @@ struct Functions
   using WorkspaceManager = typename ekat::WorkspaceManager<Spack, Device>;
   using Workspace        = typename WorkspaceManager::Workspace;
 
+  // Structure to store p3 runtime options
+  struct P3Runtime {
+    P3Runtime() = default;
```

Comment:
FYI, this kind of defaulted constructor for POD types is a bit of an overkill. In general, if a struct has only built in types, or very well known types (like std::string) that are known to be default constructible, you can omit the constructor.